### PR TITLE
[cesium] add constructor and removeProperty to PropertyBag

### PR DESCRIPTION
**Pull request by fc**
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:  https://cesium.com/docs/cesiumjs-ref-doc/PropertyBag.html and 
 https://github.com/CesiumGS/cesium/blob/1.67/Source/DataSources/PropertyBag.js#L18

@Crocsx ^

### CODE REVIEW

**Comment by fc** [comment]
@Zuzon @hnipps @szechyjs @golyalpha @KeyboardSounds @soyarsauce @jrafidi @morgansierrasnyder @Crocsx send this PR some love 😄

File: types/cesium/index.d.ts [hunk 1]
```diff
@@ -121,11 +121,14 @@ declare namespace Cesium {
         readonly isConstant: boolean;
         propertyNames: string[];
 
+        constructor(value: object, createPropertyCallback?: (value?: any) => void);
```
Comment (jrafidi) [inline types/cesium/index.d.ts]: Based on https://cesium.com/docs/cesiumjs-ref-doc/PropertyBag.html?classFilter=propertybag#PropertyBag, value is also optional

**Review by jrafidi — CHANGES_REQUESTED** [review]

File: types/cesium/index.d.ts (hunk 1 above)
Comment (fc) [reply types/cesium/index.d.ts]: thanks @jrafidi

**Review by jrafidi — APPROVED** [review]

**Comment by Crocsx** [comment]
Thanks, my bad

**Review by Crocsx — APPROVED** [review]

File: types/cesium/index.d.ts [hunk 2]
```diff
@@ -121,11 +121,14 @@ declare namespace Cesium {
         readonly isConstant: boolean;
         propertyNames: string[];
 
+        constructor(value?: object, createPropertyCallback?: (value?: any) => void);
```
Comment (amcasey) [inline types/cesium/index.d.ts]: Does the callback parameter need to be optional? https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/cesium/index.d.ts (hunk 2 above)
Comment (fc) [reply types/cesium/index.d.ts]: Did not know this - updated 6ada36d496ed9d93e0eeef52ee2bcc9f3305b7ec

**Review by amcasey — APPROVED** [review]
Thanks!

**Pull request merged**